app/presenters/hyku/work_show_presenter.rb

```diff
@@ -45,7 +45,8 @@ def date_published
     def date_accepted
       date = solr_document["date_accepted_dtsim"]
       return formatted_date(date) if date.present?
-      solr_document["date_accepted_tesim"]
+      return if solr_document["date_accepted_tesim"].blank?
+      solr_document["date_accepted_tesim"].map { |d| d.to_date.strftime("%Y") }
```

Comment:
should [these other date methods](https://github.com/scientist-softserv/britishlibrary/blob/main/app/presenters/hyku/work_show_presenter.rb#L39-L55) receive the same treatment?

Reply:
disregard, we should let the client dictate that.